src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -91,6 +92,24 @@ private void HandleVariableDeclaration(SyntaxNodeAnalysisContext context)
                     shouldAnalyze = IsStylePreferred(semanticModel, optionSet, state, cancellationToken);
                 }
             }
+            else if (declarationStatement.IsKind(SyntaxKind.Argument))
+            {
+                shouldAnalyze = false;
+                declaredType = null;
+
+                if ((declarationStatement as ArgumentSyntax)?.Expression is DeclarationExpressionSyntax declaration)
+                {
+                    declaredType = declaration.Type;
+
+                    shouldAnalyze = ShouldAnalyzeDeclarationExpression(declaration, semanticModel, cancellationToken);
+
+                    if (shouldAnalyze)
+                    {
+                        state = State.Generate(declarationStatement, semanticModel, optionSet, isVariableDeclarationContext: false, cancellationToken: cancellationToken);
```

Comment:
Yeah, it should be false as for ```ForEachStatement```. I forgot to change it back to false after testing. :)